source/implementations/f7/Meadow.F7/Devices/F7FeatherV2.cs

```diff
@@ -39,6 +39,12 @@ public F7FeatherV2()
             });
 
             Pins.Controller = this;
+
+            if (PlatformOS.SelectedNetwork == IPlatformOS.NetworkConnectionType.Cell)
+            {
+                Resolver.Log.Info($"Device is configured to use Cell for the network interface");
+                networkAdapters.Add(new CellNetworkAdapter());
```

Comment:
Do we need this, since it gets added in the NativeNetworkAdapterCollection?  Also, what if we're using a Core Compute Module?

Reply:
I think that it answers my previous question, I think that I'll remove this from here, and let it only in the `NativeNetworkAdapterCollection`.